### ISSUE TITLE
Request for option to print/return only the hash value

### ISSUE THREAD

**Issue by rhash** [rhash/RHash#259]
### Discussed in https://github.com/rhash/RHash/discussions/217

<div type='discussions-op-text'>

<sup>Originally posted by **JavascriptCockup** September 19, 2022</sup>
As title, would like an even simpler version of the --simple output format option, where only the actual hash value is returned - no filename or other information.
</div>

<br/>
I think good name for the command line option would be one of: --no-filename, --no-path, --only-hash.

For quick workarround see @greisch answer in #217.

**Comment by rhash** [comment]
Now formatting option `--one-hash` prints the selected hash and new line (LF symbol).

```sh
$ rhash --one-hash --md5 --message "abc"
900150983cd24fb0d6963f7d28e17f72

```

**Comment by rhash** [comment]
The feature is included into [RHash v1.4.5](https://github.com/rhash/RHash/releases/tag/v1.4.5).
